compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+import argparse
+
+
+def _run(cmd, is_shell=False):
+    result = subprocess.Popen(
+        cmd, shell=is_shell, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    stdout, stderr = result.communicate()
+
+    if stderr:
+        print(stderr.decode('utf-8'))
+        exit(1)
+    else:
+        return stdout
+
+
+def _image_exists(name):
+    cmd = ['docker', 'images', '-q', name]
+    lines = _run(cmd).splitlines()
+    return lines
+
+
+def main():
+    script_path = os.path.dirname(os.path.realpath(__file__))
+
+    TAG_URL = "https://api.github.com/repos/Samsung/ONE/releases/latest"
+    try:
+        response = requests.get(TAG_URL)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        raise SystemExit(e)
```

Comment:
Could you user-friendly print an error message?

`SystemExit('blah blah blah')`

Reply:
When error occured in requests for example "API rate limit exceeded for 211.187.252.208. (But here\'s the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details." message is printed to bash with 403 error code. (Git api send this message from server)
Should I print more information except this?

@jyoungyun

Reply:
Yes. I think the message is not friendly to the users. :) 
You can see some error messages in https://github.com/Samsung/ONE/blob/master/compiler/one-cmds/utils.py file.

Reply:
Thank you for advice! I'll check utils.py.